python/pyspark/pandas/tests/test_namespace.py

```diff
@@ -334,19 +334,21 @@ def test_concat_index_axis(self):
             ([psdf.reset_index(), psdf], [pdf.reset_index(), pdf]),
             ([psdf, psdf[["C", "A"]]], [pdf, pdf[["C", "A"]]]),
             ([psdf[["C", "A"]], psdf], [pdf[["C", "A"]], pdf]),
-            # only one Series
-            ([psdf, psdf["C"]], [pdf, pdf["C"]]),
-            ([psdf["C"], psdf], [pdf["C"], pdf]),
```

Comment:
nit. I believe we can keep the test coverage to prevent a future regression at `Series.concat`.

Reply:
Thanks for reivew.

Yes, actually I also moved this to L347-L348, that means we will always check all case with latest pandas, to avoid regression. I will also bump infra pandas version to 1.4.3 after all fixes complete.

For pandas<1.4.3, these two cases will failed because pandas on Spark only follow the latest pandas behaviors, so I just skip them.

If you have any other concern, feel free to comments. Thanks!

Reply:
Okay, fiar enough.

Reply:
Perfect. Thanks.